### DISCUSSION TITLE
Sprint 5 Demo

### DISCUSSION THREAD

**Discussion by Coffee2Bits** [DevOps-MBSE/AaC#163] [Archive]
Here's the plan for the Sprint 5 demo:

@Coffee2Bits 
* Intro/Sprint status
* Integrated plugins as 1st party plugins

@BryanBailey11 
* Updates to the pipeline
  * New gen-plugin smoke test
  * Reusable pipeline workflows

@cvchaparro 
* Required arrays must have at least one entry
* Self-documenting AaC DSL

**Comment by lizzcondrey** [comment]
Closing since this is marked as resolved and the sprint is past.
